readyplayerme/rpm-unity-sdk-webview
# [SDK-76] Add account linking functionality

## [SDK-76](https://ready-player-me.monday.com/boards/2563815861/pulses/SDK-76)

## Description

-   This PR adds the functionality required to pass the login token into webview setup functions and build a URL with the logintoken param (if provided
- this enables devs to trigger auto login, they just need to implement the process of requesting this token from there back end (it can't be done client side) 




## Changes

#### Added

- loginToken param to webview setup function
- login token param to BuildURL process
- ReloadUrlWithToken to trigger reload without reinitialization.



## How to Test

-   Run webview on a device and check logs for authorized event after logging in 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.







[SDK-76]: https://ready-player-me.atlassian.net/browse/SDK-76?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
